tests/utils/test_inference_endpoint.py

```
@@ -0,0 +1,314 @@
+        "        return None\n"
+        "sys.meta_path.insert(0, _Blocker())\n"
+    )
+    script = (
+        f"import sys; sys.path.insert(0, {str(tmp_path)!r}); import block_dynamo\n"
+        f"sys.path.insert(0, {str(_REPO_ROOT)!r})\n"
+        "import tests.deploy.conftest\n"
+        "import tests.deploy.test_deploy\n"
+        "import tests.deploy.test_dynamocheckpoint\n"
+        "import tests.utils.verification\n"
+    )
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)